rand_distr/src/gamma.rs

```diff
@@ -618,15 +619,15 @@ where
         if a > F::one() {
             // Algorithm BB
             let alpha = a + b;
-            let beta = ((alpha - F::from(2.).unwrap())
-                        / (F::from(2.).unwrap()*a*b - alpha)).sqrt();
+            let beta =
+                ((alpha - F::from(2.).unwrap()) / (F::from(2.).unwrap() * a * b - alpha)).sqrt();
```

Comment:
Regrettable... but probably best just to accept the result.

Reply:
We could introduce two separate variables (e.g. `beta_numer` and `beta_denom`) to make this code a bit more readable.

Reply:
Yes, but that isn't necessarily more readable. Sorry, I shouldn't have mentioned this.

Reply:
I think this is easier to read:
```
let beta_numer = alpha - F::from(2.).unwrap();
let beta_denom = F::from(2.).unwrap() * a * b - alpha;
let beta = (beta_numer / beta_denom).sqrt();
```
The current code has too many brackets. For example, at first I thought that `sqrt` applies to the denominator part only.